ppr-ui/src/components/parties/composables/useSecuredParty.ts

```diff
@@ -164,11 +158,18 @@ export const useSecuredParty = (props, context) => {
     setAddSecuredPartiesAndDebtors(parties)
   }
 
-  const addSecuredParty = (newParty: PartyIF) => {
-    let parties = getAddSecuredPartiesAndDebtors.value // eslint-disable-line
-    let newList: PartyIF[] = parties.securedParties // eslint-disable-line
-    newParty.action = ActionTypes.ADDED
-    newList.push(newParty)
+  const addSecuredParty = (newParty: PartyIF, activeIndex: number = -1) => {
+    const parties = getAddSecuredPartiesAndDebtors.value // eslint-disable-line
```

Comment:
Is there a `Parties` interface? 
If so, you could use and remove the lint suppression maybe?

Reply:
Not at a base interface level...